src/styles/utils/variables.scss

```diff
@@ -0,0 +1,9 @@
+$info-color: #616070;
+$review-color: #060b35;
+$border-color: #f3f3f3;
+$white-color: #fff;
+$star-size: 16px;
+$background-color-page: grey;
+$blue-color: #00acdc;
```

Comment:
Variable name should describe the purpose, as if this color in palette changes to red for example you have to change the name in all places where you ue it